src/framework/components/element/text-element.js

```diff
@@ -1209,7 +1209,7 @@ Object.defineProperty(TextElement.prototype, "text", {
     },
     set: function (value) {
         this._i18nKey = null;
-        var str = value && value.toString() || "";
+        var str = value != null && value.toString() || "";
```

Comment:
Personally, I'd do:
```
var str = (value == null) ? "" : value.toString();
```

See [here](https://stackoverflow.com/questions/2647867/how-can-i-determine-if-a-variable-is-undefined-or-null) for reference.